apps/ledger-live-mobile/src/screens/Swap/Form/index.tsx

```diff
@@ -46,13 +46,13 @@ import {
   updateRateAction,
   updateTransactionAction,
 } from "../../../actions/swap";
-import { TrackScreen, track, useAnalytics } from "../../../analytics";
+import { TrackScreen, useAnalytics } from "../../../analytics";
 import { Loading } from "../Loading";
 import { NotAvailable } from "./NotAvailable";
 import { TxForm } from "./TxForm";
 import { Summary } from "./Summary";
 import { Requirement } from "./Requirement";
-import { trackSwapError, SWAP_VERSION } from "../utils";
+import { SWAP_VERSION, sharedSwapTracking, useTrackSwapError } from "../utils";
```

Comment:
Can we remove SWAP_VERSION and use sharedSwapTracking across all this file?

Reply:
We still need it as it's passed to the `TrackScreen` component as a prop

Reply:
Can we use "...sharedSwapTracking"?

Reply:
https://github.com/LedgerHQ/ledger-live/pull/2275/commits/edb0317898063389a1913690ac7923d88944c5a5